lib/segment/Cargo.toml

```diff
@@ -88,6 +88,8 @@ zerocopy = { workspace = true }
 sysinfo = "0.32"
 charabia = { version = "0.9.1", default-features = false, features = ["greek", "hebrew", "thai"] }
 
+blob_store = { git = "https://github.com/qdrant/mmap-payload-storage" }
```

Comment:
we just should lock the revision like we do for the `wal` crate

Reply:
https://github.com/qdrant/qdrant/pull/5398/commits/8f697c26c0e14aa7c417c58c1bcee5e260f11431